tests/test_comments.py

```diff
@@ -72,3 +72,56 @@ def test_incorrect_fixture_aggressive(absolute_path):
         assert b'# typed_property: int = 10' in stdout
     assert b'# def function_name():' in stdout
     assert b'# class CommentedClass(object):' in stdout
+
+
+def test_incorrect_fixture_whitelist(absolute_path):
+    """End-to-End test to check that incorrect code raises warning."""
+    filename = absolute_path('fixtures', 'incorrect.py')
+    process = subprocess.Popen(
+        [
+            'flake8',
+            '--eradicate-whitelist',
+            'just#overwrite',
```

Comment:
can we make this a `list`?

Reply:
What specifically do you want to be a list? The `'just#overwrite'` string? If so thats not possible with the current version of eradicate, as how I implemented the whitelist feature.

Reason for this is, this way all white-spaces are preserved the user puts in front or after the `#`.